features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -257,6 +274,9 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
             if (isFeatureEnabled(featureConfig?.applications,
                 ApplicationManagementConstants.FEATURE_DICTIONARY.get("APPLICATION_EDIT_ACCESS_CONFIG"))
                 && !isFragmentApp
+                && (legacyAuthzRuntime ||
+                        !(application?.clientId === ApplicationManagementConstants.MY_ACCOUNT_CLIENT_ID
```

Comment:
Let's define a variable for this in the top of the component definition, and reuse it in other places for readability.

eg:

```js
const areSimplifiedSaaSMyAccountSettingsEnabled: boolean = application?.clientId === ApplicationManagementConstants.MY_ACCOUNT_CLIENT_ID
                            && myAccountSimplifiedSettings
```